app/models/article.rb

```diff
@@ -41,13 +41,8 @@ def more_link
   private
 
   # :reek:FeatureEnvy
-  def handle_summon_error(error)
-    message = if error.is_a? Summon::Transport::AuthorizationError
-                'Could not authenticate to the upstream Summon service'
-              else
-                error.message
-              end
+  def handle_summon_authorization_error
```

Comment:
This method is only called in cases when there is a `Summon::Transport::TransportError`, so there is no need for an additional type check to confirm that it is a `Summon::Transport::TransportError`, or logic in the `else` branch in case it is not a `Summon::Transport::TransportError`.